apps/server/package.json

```diff
@@ -1,16 +1,30 @@
 {
   "name": "@react-native-templates/rise-server",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ericvicenti/react-native-templates.git",
+    "directory": "apps/server"
+  },
+  "author": "Eric Vicenti <ericvicenti@gmail.com> (https://github.com/ericvicenti)",
+  "contributors": [
+    "Mike Grabowski <grabbou@gmail.com> (https://github.com/grabbou)"
+  ],
+  "bugs": {
+    "url": "https://github.com/ericvicenti/react-native-templates/issues"
+  },
+  "homepage": "https://github.com/ericvicenti/react-native-templates#readme",
   "scripts": {
-    "start": "esr server.ts",
-    "start:demo": "nodemon -e js,ts,tsx --exec \"esr demo-server.ts\"",
-    "start:dev": "nodemon -e js,ts,tsx",
-    "convert-videos": "esr eg-video-conversion.ts"
+    "start": "tsx server.ts",
```

Comment:
this will be slower than `esbuild runner` but no big deal

Reply:
Hm, I haven't run benchmarking, but it has better interior with TypeScript, so gets all the `tsconfigs` right. I was thinking that for production deployments, we should run `tsc` and run the server with pure `node` instead.